src/app/components/client/ExposuresFilter.tsx

```diff
@@ -249,22 +249,24 @@ export const ExposuresFilter = ({
           <li className={`${styles.hideOnMobile} ${styles.companyNameArea}`}>
             {l10n.getString("dashboard-exposures-filter-company")}
           </li>
-          <li className={styles.hideOnMobile}>
-            {l10n.getString("dashboard-exposures-filter-exposure-type")}
-            {isEligibleForPremium && (
-              <button
-                {...exposureTypeExplainerTriggerProps}
-                ref={exposureTypeExplainerTriggerRef}
-                aria-label={l10n.getString("modal-open-alt")}
-              >
-                <QuestionMarkCircle
-                  width="15"
-                  height="15"
-                  alt={l10n.getString("modal-open-alt")}
-                />
-              </button>
-            )}
-          </li>
+          {isEligibleForPremium && (
+            <li className={styles.hideOnMobile}>
+              {l10n.getString("dashboard-exposures-filter-exposure-type")}
+              {isEligibleForPremium && (
+                <button
+                  {...exposureTypeExplainerTriggerProps}
+                  ref={exposureTypeExplainerTriggerRef}
+                  aria-label={l10n.getString("modal-open-alt")}
+                >
+                  <QuestionMarkCircle
+                    width="15"
+                    height="15"
+                    alt={l10n.getString("modal-open-alt")}
+                  />
```

Comment:
Unrelated to this PR, but was an interesting grep for all usages of `<QuestionMarkCircle>`:
```sh
git grep -n -A9 "<QuestionMarkCircle" | grep -E "(QuestionMarkCircle|width=|height=|alt=)"

src/app/(proper_react)/redesign/(public)/PlansTable.tsx:1013:        <QuestionMarkCircle
src/app/(proper_react)/redesign/(public)/PlansTable.tsx-1014-          alt={l10n.getString(
src/app/components/client/Chart.tsx:320:            <QuestionMarkCircle
src/app/components/client/Chart.tsx-321-              width="15"
src/app/components/client/Chart.tsx-322-              height="15"
src/app/components/client/Chart.tsx-323-              alt={l10n.getString("modal-open-alt")}
src/app/components/client/ExposureCard.tsx:330:          icon={<QuestionMarkCircle alt="" width="13" height="13" />}
src/app/components/client/ExposureCard.tsx:344:          icon={<QuestionMarkCircle alt="" width="13" height="13" />} // default icon for categories without a unique one
src/app/components/client/ExposuresFilter.tsx:261:                  <QuestionMarkCircle
src/app/components/client/ExposuresFilter.tsx-262-                    width="15"
src/app/components/client/ExposuresFilter.tsx-263-                    height="15"
src/app/components/client/ExposuresFilter.tsx-264-                    alt={l10n.getString("modal-open-alt")}
src/app/components/client/ExposuresFilter.tsx:280:              <QuestionMarkCircle
src/app/components/client/ExposuresFilter.tsx-281-                width="15"
src/app/components/client/ExposuresFilter.tsx-282-                height="15"
src/app/components/client/ExposuresFilter.tsx-283-                alt={l10n.getString("modal-open-alt")}
```

Specifies `alt=""` but not a `width` or `height`:
https://github.com/mozilla/blurts-server/blob/29133cd3901ea8a7232bb27c179c93b92f23ef2f/src/app/(proper_react)/redesign/(public)/PlansTable.tsx#L1013-L1017

Empty `alt=""` (which might be fine since it might just be an informational icon and not a modal toggle):
https://github.com/mozilla/blurts-server/blob/29133cd3901ea8a7232bb27c179c93b92f23ef2f/src/app/components/client/ExposureCard.tsx#L317
https://github.com/mozilla/blurts-server/blob/29133cd3901ea8a7232bb27c179c93b92f23ef2f/src/app/components/client/ExposureCard.tsx#L328